__tests__/utils.test.js

```diff
@@ -148,15 +149,47 @@ describe("GET /api", () => {
       .expect(200)
       .then((response) => {
         const data = response.body;
-        const output = {
-          "GET /api":
-            '{"description":"serves up a json representation of all the available endpoints of the api"}',
-          "GET /api/topics":
-            '{"description":"serves an array of all topics","queries":[],"exampleResponse":{"topics":[{"slug":"football","description":"Footie!"}]}}',
-          "GET /api/articles":
-            '{"description":"serves an array of all articles","queries":["author","topic","sort_by","order"],"exampleResponse":{"articles":[{"title":"Seafood substitutions are increasing","topic":"cooking","author":"weegembump","body":"Text from the article..","created_at":"2018-05-30T15:59:13.341Z","votes":0,"comment_count":6}]}}',
-        };
-        expect(data).toEqual(output);
+        expect(data).toEqual(endpointFile);
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("Should return the correct article information when passed an article id", () => {
+    return request(app)
+      .get("/api/articles/4")
+      .expect(200)
+      .then((response) => {
+        const data = response.body.articleData;
+        expect(data.article_id).toBe(4);
+        expect(data.hasOwnProperty("article_id")).toBe(true);
```

Comment:
We have requested a specific article so we should check in our test that the correct article is returned therfore we should put the actual id here